crates/locutus-node/src/operations/subscribe.rs

```diff
@@ -469,6 +472,12 @@ mod messages {
         pub fn sender(&self) -> Option<&PeerKeyLocation> {
             None
         }
+
+        pub fn target(&self) -> Option<&PeerKeyLocation> {
+            match self {
+                _ => None,
```

Comment:
I guess this is temporal  right? Don't forget to change it.

Reply:
i updated it in this branch, but the change has not been updated?